integration/src/test/java/org/kie/hacep/PodAsLeaderTest.java

```diff
@@ -131,8 +131,7 @@ public void processOneSentMessageAsLeaderTest() {
             assertEquals(insert.getKey(),eventsRecordTwo.key());
             assertTrue(!insert.getSideEffects().isEmpty());
         } catch (Exception ex) {
-            logger.error(ex.getMessage(),
-                         ex);
+            logger.error(ex.getMessage(), ex);
```

Comment:
Please throw exception and allow test to fail fast.

Reply:
Ok